docs/dm/attestation-results/attestation-results.cddl

```diff
@@ -0,0 +1,44 @@
+start = attestation-results
+
+$ar-status /= ar.status-failure
+$ar-status /= ar.status-success
+$ar-status /= ar.status-unknown
+
+attestation-results = {
+  ar.status => $ar-status
+  ? ar.trust-vector => ar-trust-vector
+  ? ar.raw-evidence => ar-bytes
+  ar.timestamp => ar-timestamp
+  ? ar.endorsed-claims => ar-endorsed-claims
+  * $$attestation-results-extension
+}
+
+ar-trust-vector = {
+  ar.hw-authenticity => $ar-status
+  ar.sw-integrity => $ar-status
+  ar.sw-up-to-dateness => $ar-status
+  ar.config-integrity => $ar-status
+  ar.runtime-integrity => $ar-status
+  ar.certification-status => $ar-status
+  * $$ar-trust-vector-extension
+}
+
+ar-endorsed-claims = non-empty<{
+  ? ar.hw-details => ar-hw-details
+  ? ar.sw-details => ar-sw-details
+  ? ar.certification-details => ar-certification-details
+}>
+
+ar-hw-details = {
+  + ar-label => any
+}
+
+ar-sw-details = {
+  + ar-label => any
+}
+
```

Comment:
Should we also have a config-details section here! I recollect in many attestation use cases, configuration becomes a key element of Attestation ?

Reply:
I am not sure that configuration endorsements would fit in to the SW category.  It looks to me that if we want to have them, we should make a placeholder in the ar-endorsed-claims level?

Reply:
Yes I meant at endorsed level only.

I mean at ar-endorsed-claims = non-empty<{
? ar.config-details => ar-config-details
}

and then expand the config. like you did for other details ?

Reply:
> Yes I meant at endorsed level only.

Cool, makes sense. I'll add it.

Reply:
Thank You!

Reply:
It's possible that config items will fit into the same bucket as software items e.g. the config item is likely to be a signed blob which can be measured as loaded. Only a description / name would distinguish from a similarly distributed excutable component

Reply:
As I understand, TCG and other specifications clearly see a difference in SW and Config. Better to keep them separate, though I agree the actual signed blob may look very similar semantically and the steps to verify may also be similar